app/models/profile.rb

```diff
@@ -41,10 +40,23 @@ def activate
   end
   # rubocop:enable Rails/SkipsModelValidations
 
+  def activate_after_passing_review
+    update!(fraud_review_pending: false)
+    activate
+  end
+
   def deactivate(reason)
     update!(active: false, deactivation_reason: reason)
   end
 
+  def deactivate_for_fraud_review
+    update!(active: false, fraud_review_pending: true)
```

Comment:
For completeness, should this set `fraud_rejection` to `false`? Maybe someone can get re-reviewed. Possibly also in the method above, `activate_after_passing_review`.

Reply:
I'm wondering if what we actually want is a single column with an enum? 

```fraud_status = :in_review / :rejected / :approved```

But I realize this is not what was specified in the ticket, and not looking to bikeshed this particular thing.